source/help/getting-started/updating-your-profile.rst

```diff
@@ -0,0 +1,27 @@
+=====================
+Updating your profile
+=====================
+
+Your profile includes information such as your name as it appears in Mattermost, your email address for receiving notifications and password resets, and your profile picture.
```

Comment:
Propose instead we mention what we recommend updating after account creation. In my mind that is adding a Name and profile picture.

`Your profile includes information such as your name as it appears in Mattermost, your email address for receiving notifications and password resets, and your profile picture.` > 
```
After account creation it is recommended that you update your **Full Name** and **Profile Picture** to make it easier for your teammates to find you. Update your profile by clicking your username at the top left of the screen to open the Main Menu, then click **Account Settings**.

To learn more about **Account Settings** and configuring your profile please see the following related links:

- :doc:`../settings/account-settings`
- :doc:`../settings/configuring-notifications`
```